.github/workflows/set_reviewers.yml

```
@@ -0,0 +1,22 @@
+name: Set Reviewers
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  set-reviewers:
+    if: github.repository_owner == 'viamrobotics'
+    runs-on: [self-hosted, x64]
+    container:
+      image: ghcr.io/viamrobotics/canon:amd64
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pleo-io/assign-random-codeowners@v0.2.8
+        with:
+          reviewers-to-assign: 2
+          assign-individuals-from-teams: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GIT_ACCESS_TOKEN }}
```

if you are referencing the automatically-created token that is created with each workflow run, then I think you need to use:

```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

---

this is a different token! you can see it in the settings under `Secrets and variables`/`actions` :) 

Although using `secrets.GIT_ACCESS_TOKEN` was sufficient to make the job run for `rust-sdk`, but doesn't seem to be working here... I wonder if it's because this isn't actually merged into main yet?